hubploy/auth.py

```diff
@@ -216,32 +232,45 @@ def cluster_auth_gcloud(deployment, project, cluster, zone, service_key):
     yield
 
 
-def cluster_auth_aws(deployment, project, cluster, zone, service_key):
+def cluster_auth_aws(deployment, project, cluster, zone, service_key = None, eks_role = None):
```

Comment:
pep8 recommends no spaces around `=` used to denote defaults for arguments (https://www.python.org/dev/peps/pep-0008/#other-recommendations).